packages/KTable/KTable.vue

```diff
@@ -173,22 +195,43 @@ export default {
     },
 
     trlisteners () {
-      return pluckListeners('row:', this.$listeners)
+      return (entity, type) => {
+        const pluckedListeners = pluckListeners('row:', this.$listeners)(entity, type)
+
+        return {
+          ...pluckedListeners,
+          mouseup (e) {
+            if (e.target.tagName === 'TD' && pluckedListeners['mouseup']) {
+              pluckedListeners['mouseup'](e, entity, type)
+            }
+          }
+        }
+      }
     }
   }
 }
 </script>
 
 <style scoped lang="scss">
 @import '~@kongponents/styles/_variables.scss';
-
-table.k-table {
-  border-collapse: collapse;
-}
-
 .k-table {
   width: 100%;
   max-width: 100%;
+  border-collapse: collapse;
+
+  &.side-border.has-hover {
```

Comment:
This should remain the way it was but. IF it has hover then add the hover styles (steal-300)